### PR TITLE
Added LinkMore component and link more in HeroImageLeft block

### PR DESCRIPTION
**Pull request by giuliaghisini**
Schermata 2021-10-04 alle 18 37 06
This adds fields "Link more title" and "Link more href" to the HeroImageLeft block as in ListingBlock"

### CODE REVIEW

File: src/components/manage/Blocks/HeroImageLeft/Data.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import schemaHero from './Schema';
+import { BlockDataForm } from '@plone/volto/components';
+
+const TestBlockData = (props) => {
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/HeroImageLeft/Data.jsx:6]: Should be something else, maybe?

File: src/components/manage/Blocks/HeroImageLeft/Schema.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { defineMessages } from 'react-intl';
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/HeroImageLeft/Schema.jsx:1]: I think the filename should be `schema.js`. @sneridagh?

**Review by tiberiuichim — APPROVED** [review]
Some naming, otherwise good to go.

**Review by nzambello — APPROVED** [review]
Agree with Tiberiu for the namings, but LGTM

**Comment by giuliaghisini** [comment]
> Some naming, otherwise good to go.

fixed!

**Review by sneridagh — APPROVED** [review]

**Pull request merged**